app/src/classList.json

```diff
@@ -0,0 +1,975 @@
+[

```

Comment:
You should be using the existing data instead to get your classes (from my previous PR) since that's what you'll be getting from API anyways. I assume you pulled all of this from that file, so you should just be able to add that into the `Survey.tsx` file as a hook that just extracts this bit of the total returned data object. You'll feed in the same info and we won't have to add the conversion later

Reply:
Since the existing data is outside the src folder I don't think I can access it. I made that file and trimmed it down to signify what it was for, then I was thinking we would delete it when we integrate with api